### PR TITLE
adding the frames easing function to the formal syntax... 

### PR DESCRIPTION
**Pull request by chrisdavidmills**
…of &lt;single-transition-timing-function&gt;.

As specified in https://www.w3.org/TR/css-timing-1/#frames-timing-functions

This is part of the docs for https://bugzilla.mozilla.org/show_bug.cgi?id=1248340

### CODE REVIEW

**Comment by lahmatiy** [comment]
If we following the spec, those syntaxes should be added/updated:
```
<single-timing-function> = linear | <cubic-bezier-timing-function> | <step-timing-function> | <frames-timing-function>
<cubic-bezier-timing-function> = ease | ease-in | ease-out | ease-in-out | cubic-bezier(<number>, <number>, <number>, <number>)
<step-timing-function> = step-start | step-end | steps(<integer>[, [ start | end ] ]?)
<frames-timing-function> = frames(<integer>)
```
And `single-transition-timing-function` should be changed that way:
```
<single-transition-timing-function> = <single-timing-function>
```

**Comment by Elchi3** [comment]
@chrisdavidmills Do you want to make the suggested changes?

Do we want to wait until https://github.com/w3c/csswg-drafts/issues/1301 is resolved?

**Comment by chrisdavidmills** [comment]
Changes made. Thanks!

File: css/syntaxes.json [hunk 1]
```diff
@@ -172,11 +174,12 @@
   "single-animation-fill-mode": "none | forwards | backwards | both",
   "single-animation-iteration-count": "infinite | <number>",
   "single-animation-play-state": "running | paused",
-  "single-timing-function": "<single-transition-timing-function>",
+  "single-timing-function": "linear | <cubic-bezier-timing-function> | <step-timing-function> | <frames-timing-function>",
   "single-transition": "[ none | <single-transition-property> ] || <time> || <single-transition-timing-function> || <time>",
-  "single-transition-timing-function": "ease | linear | ease-in | ease-out | ease-in-out | step-start | step-end | steps( <integer> [, [ start | end ] ]? ) | cubic-bezier( <number>, <number>, <number>, <number> )",
+  "single-transition-timing-function": "ease | linear | ease-in | ease-out | ease-in-out | step-start | step-end | frames( <integer> ) | steps( <integer> [, [ start | end ] ]? ) | cubic-bezier( <number>, <number>, <number>, <number> )",
```
Comment (Elchi3) [inline css/syntaxes.json]: This should be `single-transition-timing-function": "<single-timing-function>"` now that we defined the other syntaxes, right?

Comment (chrisdavidmills) [inline css/syntaxes.json]: Yup, think you're right. Good spot sir.

**Pull request merged**